hail/src/main/scala/is/hail/expr/ir/ExtractIntervalFilters.scala

```diff
@@ -115,7 +115,7 @@ object ExtractIntervalFilters {
 
   def canGenerateOpenInterval(t: Type): Boolean = t match {
     case _: TNumeric => true
```

Comment:
I think this can be case TNumeric

Reply:
that won't compile, TNumeric is a trait or abstract class.

Reply:
ah actually maybe this doesn't work with inheritance; I thought _: only used for differing constructors, but since TNumeric isn't concrete probably can't match on it directly